Doc/source/misc/cliTools.rst

```diff
@@ -1,6 +1,6 @@
-########
-cliTools
-########
+###################################################################
+cliTools: Utilities for command-line interfaces and console scripts
```

Comment:
I am not sure if we should advertise this module outside of fontTools, I think it's mostly for internal use. Oh well, I guess it doesn't harm.

Reply:
`makeOutputFileName` *is* used outside I believe.